<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsGlobeVectorLayerPropertiesPage</class>
 <widget class="QgsVectorLayerPropertiesPage" name="QgsGlobeVectorLayerPropertiesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="mGbxAltitude">
     <property name="title">
      <string>Altitude</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="mLblClamping">
        <property name="toolTip">
         <string>Terrain following behaviur</string>
        </property>
        <property name="text">
         <string>Clamping</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="mCbxAltitudeClamping">
        <property name="toolTip">
         <string>Terrain following behavior</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mLblTechnique">
        <property name="toolTip">
         <string>Terrain following technique</string>
        </property>
        <property name="text">
         <string>Technique</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="mLblBinding">
        <property name="toolTip">
         <string>Granulatiry at which to sample the terrain</string>
        </property>
        <property name="text">
         <string>Binding</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="mLblResolution">
        <property name="toolTip">
         <string>Elevation data resolution at which to sample terrain height</string>
        </property>
        <property name="text">
         <string>Resolution</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="mLblOffset">
        <property name="toolTip">
         <string>Vertical offset to apply to geometry Z</string>
        </property>
        <property name="text">
         <string>Offset</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="mLblScale">
        <property name="toolTip">
         <string>Scale factor to apply to geometry Z</string>
        </property>
        <property name="text">
         <string>Scale</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="mCbxAltitudeTechnique">
        <property name="toolTip">
         <string>Terrain following technique</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="mCbxAltitudeBinding">
        <property name="toolTip">
         <string>Granulatiry at which to sample the terrain</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="mTxtAltitudeResolution">
        <property name="toolTip">
         <string>Elevation data resolution at which to sample terrain height</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="mTxtAltitudeOffset">
        <property name="toolTip">
         <string>Vertical offset to apply to geometry Z</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="mTxtAltitudeScale">
        <property name="toolTip">
         <string>Scale factor to apply to geometry Z</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mGbxExtrusion">
     <property name="title">
      <string>E&amp;xtrusion</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="mLblHeight">
        <property name="text">
         <string>Height [m]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="mTxtHeight"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="mLblWallGradient">
        <property name="text">
         <string>Wall gradient</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="mTxtWallGradient">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="mChkExtrusionFlatten">
        <property name="text">
         <string>Flatten</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mGbxLabelingEnabled">
     <property name="title">
      <string>Enable &amp;labeling</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QCheckBox" name="mCbxLabelingDeclutter">
        <property name="text">
         <string>Declutter</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mChkLighting">
     <property name="text">
      <string>Lighting</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsVectorLayerPropertiesPage</class>
   <extends>QWidget</extends>
   <header>qgsvectorlayerpropertiespage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
